docs/design/proposals/rbd-pv-key-rotation.md

```diff
@@ -0,0 +1,77 @@
+# Encryption Key Rotation
+
+## Proposal
+
+Subject of this proposal is to add support for rotation of
+encryption keys (KEKs) for encrypted volumes in Ceph-CSI.
+
+Support for rotating keys on RWX/ROX volumes is out of scope for now
+and shall be added later.
+
+## Document Terminology
+
+- Encryption Key: The passphrase that is used to encrypt and open the device.
+- LUKS: The specification used by dm-crypt to process encrypted volumes on linux.
+
+## Proposed Solution
+
+The proposed solution in this document, is to address the periodic rotation
```

Comment:
How is this a proposal for `periodic` rotation?  There is no scheduler that does the key rotation at certain intervals/periods. This is a description of what the CSI-Addons API requests will do, who/whatever the API calls at what intervals/periods is out of scope for the implementation of the API.